src/net/sourceforge/kolmafia/request/CharPaneRequest.java

```diff
@@ -1559,7 +1559,11 @@ public static final void parseStatus(final JSONObject JSON) throws JSONException
     }
 
     Object lmo = JSON.get("limitmode");
-    KoLCharacter.setLimitMode(lmo.toString());
+    if (lmo instanceof Integer && lmo.equals(0)) {
+      KoLCharacter.setLimitMode(LimitMode.NONE);
+    } else {
+      KoLCharacter.setLimitMode(lmo.toString());
```

Comment:
We should probably instanceof string check here and error out otherwise. And add a coverage test to this slice of the code too, why not

Reply:
aye, will merge here and follow-up.